README.md

```diff
@@ -125,15 +125,15 @@ pre-commit:
 ## Custom file list
 
 Lefthook can be even more specific in selecting files.
-If you want to choose diff of all changed files between the current branch and master branch you can do it this way:
+If you want to choose all files changed locally but pushed to GitHub yet.
 
 ```yml
 # lefthook.yml
 
 pre-push:
   commands:
     frontend-style:
-      files: git diff --name-only master # custom list of files
+      files: git diff --name-only origin # custom list of files
```

Comment:
I think this command is the same as the previous one. How about `git diff @{upstream}`?

Reply:
`master` === local master. `origin` means GitHub.

I didn’t test `@{upstream}` (and remove Lefhook) can you test it?

Reply:
Sure!

Reply:
Command `git diff --name-only origin` return a error.
I suppose `git diff --name-only @{push}` enough for that kind example?